### ISSUE TITLE
request: Voice message

### ISSUE THREAD

**Issue by GeekCornerGH** [skyra-project/discord-components#430]
### Is there an existing issue or pull request for this?

- [X] I have searched the existing issues and pull requests

### Feature description

I just want to have voice messages

### Desired solution

A discord-voice-message component or something like that

### Alternatives considered

None

**Comment by favna** [comment]
Mind you that at best it could be a variation to the audio component and reference a pre-existing audio file, because as a webcomponent I am not going to provide a service for storing mp3 and prompting people to enable their mic to record an actual voice clip.

**Comment by GeekCornerGH** [comment]
Of course, making an host is out of the scope of this project. Usescases for this would be ticket transcripts for example instead

**Comment by mezotv** [comment]
> Of course, making an host is out of the scope of this project. Usescases for this would be ticket transcripts for example instead

How would this even work and more importantly how would the bars and all be picked?

**Comment by favna** [comment]
@mezotv The same way the `discord-audio-attachment` works. Lots of custom CSS and using the native `audio` element.
